Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
Maybe this is better at the start of MGCB.exe ?  Then it will report this issue if it is run from the Pipeline tool, the command line, Visual Studio, etc.

Maybe consider adding the same line to 2MGFX.exe as well?